external/anomaly/tasks/inference.py

```diff
@@ -291,8 +291,11 @@ def save_model(self, output_model: ModelEntity) -> None:
     def _set_metadata(self, output_model: ModelEntity):
         output_model.set_data("image_threshold", self.model.image_threshold.value.cpu().numpy().tobytes())
         output_model.set_data("pixel_threshold", self.model.pixel_threshold.value.cpu().numpy().tobytes())
-        output_model.set_data("min", self.model.min_max.min.cpu().numpy().tobytes())
-        output_model.set_data("max", self.model.min_max.max.cpu().numpy().tobytes())
+        if hasattr(self.model, "min_max"):
+            output_model.set_data("min", self.model.min_max.min.cpu().numpy().tobytes())
+            output_model.set_data("max", self.model.min_max.max.cpu().numpy().tobytes())
+        else:
+            logger.warning("Model saved without min_max")
```

Comment:
Maybe we should mention that model wasn't initialized properly.

Reply:
I think no, because a model was initialized properly. It happens if save a model without running training pipeline. 
Can you suggest more correctly warning message?

Reply:
Min-Max is assigned when `setup` is called. It is initialized when the training pipeline is run. If we just create a model and do not call the train loop, it will in consequence not initialize the required classes used for normalization. The output produced in this case will not be good.

Reply:
Maybe we can say something like `the model was not trained before saving. This will lead to incorrect normalization of the heatmaps.`

Reply:
Thanks